.gitignore

```diff
@@ -1,6 +1,5 @@
 # composer
 /composer.phar
-/composer.lock
```

Comment:
I think you can keep this one there. This is just a tooling package, so maintaining dependencies are extra work that will not pay off.

Also, the upgrade tool should always be tested on bleeding edge versions, so the upgrade to the latest version works :)